### PR TITLE
Add Slack information as a way to contact oneDPL developers.

### PR DESCRIPTION
**Pull request by timmiesmith**
This PR addresses the following items in the oneDPL migration to UXL Foundation governance:
https://github.com/uxlfoundation/open-source-working-group/issues/6
https://github.com/uxlfoundation/open-source-working-group/issues/11
https://github.com/uxlfoundation/open-source-working-group/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ You can find oneDPL samples at the [oneDPL Samples](https://github.com/oneapi-sr
 ## Support and Contribution
 Please report issues and suggestions via [GitHub issues](https://github.com/oneapi-src/oneDPL/issues).
 
+You can also contact oneDPL developers via [UXL Foundation Slack](https://slack-invite.uxlfoundation.org/) using
```
Comment (danhoeflinger) [inline README.md:40]: To satisfy https://github.com/uxlfoundation/open-source-working-group/issues/6 , do we need to name the specific people who are monitoring the channel here in this PR, or is it enough to have assigned this?  It seems we may need to name the people here.

Comment (rodburns) [reply README.md:40]: Ideally there is a table similar to the one here listing resposible maintainers for different parts of the project. https://github.com/oneapi-src/oneDNN/blob/main/MAINTAINERS.md

You don't need to say who is monitoring the Slack specifically so long as someone is checking.

**Review by danhoeflinger — APPROVED** [review]
LGTM

**Comment by timmiesmith** [comment]
@danhoeflinger sorry, I seem to have made a mess of this PR.  Let me see if I can resolve it or need to open a new PR.

**Comment by timmiesmith** [comment]
> @danhoeflinger sorry, I seem to have made a mess of this PR. Let me see if I can resolve it or need to open a new PR.

Branch update issues have been resolved.  One final review would be appreciated.  Thank you.

**Review by danhoeflinger — APPROVED** [review]
LGTM

**Comment by vbm23** [comment]
@ValentinaKats Requesting your review for this PR

**Pull request merged**